subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -12,6 +12,15 @@ menuconfig BT_FAST_PAIR
 
 if BT_FAST_PAIR
 
+config BT_FAST_PAIR_SUBSEQUENT_PAIRING
+	bool "Fast Pair Subsequent Pairing"
+	default y if !BT_FAST_PAIR_FMDN
```

Comment:
I think we should not allow enabling it for `CONFIG_BT_FAST_PAIR_FMDN`. The FMDN supports only minimal AK backend so we cannot pass subsequent pairing anyway

Reply:
well, this is not true, reverting the change

Reply:
Ahh, you are right. The subsequent pairing does not write an additional Account Key.
Please disregard this comment.